tests/@angular-devkit/build-optimizer/webpack/aio-app/src/app/app.component.ts

```diff
@@ -0,0 +1,338 @@
+  currentDocVersion: NavigationNode;
+  currentNodes: CurrentNodes;
+  currentPath: string;
+  docVersions: NavigationNode[];
+  dtOn = false;
+  footerNodes: NavigationNode[];
+
+  /**
+   * An HTML friendly identifier for the currently displayed page.
+   * This is computed from the `currentDocument.id` by replacing `/` with `-`
+   */
+  pageId: string;
+  /**
+   * An HTML friendly identifer for the "folder" of the currently displayed page.
+   * This is computed by taking everything up to the first `/` in the `currentDocument.id`
+   */
+  folderId: string;
+  /**
+   * These CSS classes are computed from the current state of the application
+   * (e.g. what document is being viewed) to allow for fine grain control over
+   * the styling of individual pages.
+   * You will get three classes:
+   *
+   * * `page-...`: computed from the current document id (e.g. news, guide-security, tutorial-toh-pt2)
+   * * `folder-...`: computed from the top level folder for an id (e.g. guide, tutorial, etc)
+   * * `view-...`: computef from the navigation view (e.g. SideNav, TopBar, etc)
+   */
+  @HostBinding('class')
+  hostClasses = '';
+
+  isFetching = false;
+  isStarting = true;
+  isSideBySide = false;
+  private isFetchingTimeout: any;
+  private isSideNavDoc = false;
+
+  private sideBySideWidth = 992;
+  sideNavNodes: NavigationNode[];
+  topMenuNodes: NavigationNode[];
+  topMenuNarrowNodes: NavigationNode[];
+
+  hasFloatingToc = true;
+  private showFloatingToc = new BehaviorSubject(false);
+  private showFloatingTocWidth = 800;
+  tocMaxHeight: string;
+  private tocMaxHeightOffset = 0;
+
+  versionInfo: VersionInfo;
+
+  get homeImageUrl() {
+    return '';
+    // return this.isSideBySide ?
+    //   'assets/images/logos/standard/logo-nav@2x.png' :
+    //   'assets/images/logos/standard/shield-large.svg';
+  }
+  get isOpened() { return this.isSideBySide && this.isSideNavDoc; }
+  get mode() { return this.isSideBySide ? 'side' : 'over'; }
+
+  // Need the doc-viewer element for scrolling the contents
+  @ViewChild(DocViewerComponent, { read: ElementRef })
+  docViewer: ElementRef;
+
+  // Search related properties
+  showSearchResults = false;
+  @ViewChildren('searchBox, searchResults', { read: ElementRef })
+  searchElements: QueryList<ElementRef>;
+  @ViewChild(SearchResultsComponent)
+  searchResults: SearchResultsComponent;
+  @ViewChild(SearchBoxComponent)
+  searchBox: SearchBoxComponent;
+
+  @ViewChild(MdSidenav)
+  sidenav: MdSidenav;
+
+  constructor(
+    private documentService: DocumentService,
+    private hostElement: ElementRef,
+    private locationService: LocationService,
+    private navigationService: NavigationService,
+    private scrollService: ScrollService,
+    private searchService: SearchService,
+    private swUpdateNotifications: SwUpdateNotificationsService,
+    private tocService: TocService
+  ) { }
+
+  ngOnInit() {
+    // Do not initialize the search on browsers that lack web worker support
```

Comment:
Is that code supposed to be in comments? If so, please add a comment explaining that this comment is for a specific test.